libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -42,6 +42,7 @@ export const lanaLog = ({ message, e = '', tags = 'errorType=default' }) => {
     sampleRate: 1,
     tags,
   });
+  console.error(message);
```

Comment:
Why would we show a console error ?

Reply:
Now that we have non-Milo consumers as well, it seems appropriate to display the error in the console